both/collections/competition.js

```diff
@@ -100,9 +120,7 @@ Competitions.attachSchema({
         return "missingRegistrationOpenDate";
       }
 
-      if(registrationOpenDate.getTime() < registrationCloseDate.getTime()) {
-        return null;
-      } else {
+      if(registrationOpenDate.getTime() >= registrationCloseDate.getTime()) {
```

Comment:
Might be a bit odd that this function doesn't use the `validationObject` method you introduced above. Doesn't seem worth changing, though.